terraform/modules/rds-aurora-database/main.tf

```diff
@@ -0,0 +1,99 @@
+// setting the provider
+provider "aws" {
```

Comment:
I think you can remove this. We can set the provider when calling the modules.

Reply:
of course fabio, I just put it there so it could be 'runnable'

Reply:
@feernandobraga, I usually create a folder in `terraform/examples/my-testing-module` what will be the entry point to call the module that I am working on.